connector/pom.xml

```diff
@@ -74,7 +74,7 @@
         <dependency>
             <groupId>com.nukkitx.protocol</groupId>
             <artifactId>bedrock-v361</artifactId>
-            <version>2.1.2</version>
+            <version>2.1.3</version>
```

Comment:
Btw 2.2.0 is out

Reply:
I know, hopefully I'll update and pull it it today